alws/crud/build_node.py

```diff
@@ -565,11 +556,23 @@ async def build_done(db: Session, pulp: PulpClient, request: build_node_schema.B
     db_srpm_name = None
     srpm = None
     for rpm in rpms_result.scalars().all():
+        # if build task is excluded or failed, we don't need to create
+        # SourceRpm and BinaryRpm records, because it's break package releases
+        if status in (BuildTaskStatus.EXCLUDED, BuildTaskStatus.FAILED):
```

Comment:
So you will not produce SourceRpm instance, but build artifact will be there, correct?

Reply:
That’s correct